src/css/services.css

```diff
@@ -0,0 +1,64 @@
+.services_heading {
+    display: flex;
+    justify-content: center;
+    font-weight: bolder;
+    font-family: 'Maven Pro', sans-serif;
+    font-size: 55px;
+    margin-top: 40px;
+    margin-bottom: 40px;
+    }
+    
+    .services_box {
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        align-items: center;
+    }
+    
+    
+    .services_picture {
+        width: 100%;
+        height: 200px;
+        object-fit: cover;
+        display: block;
+    }
+    
+    .service_card_container {
+        display: flex;
+        flex-direction: column;
+        margin: 26px;
+        margin-top: 10px;
+        transition: 0.2s;
+        background-color: rgba(158, 163, 168, 0.692);
+        border-radius: 6px;
+        overflow: hidden;
+        box-shadow: 1px 1px 10px black;
+    }
+    
+    .service_card_container:hover {
+        transform: scale(1.05) 
+    } 
+    
+    .service_lists {
+        list-style-type: disc;
+        max-width: 300px;
+        max-height: 600px;
+        margin: 10px 30px 8px;
+        padding: 20;
```

Comment:
padding is missing pixels. This might not even be applying - in which case you want to remove it.